### PR TITLE
Created todo with next and TS

### PR DESCRIPTION
**Pull request by lovedeep5**
Run the app using "npm run dev", it will spin up the app and will show a link only, click on the link to view the component.

### CODE REVIEW

File: src/app/page.tsx [hunk 1]
```diff
@@ -1,95 +1,16 @@
-import Image from 'next/image'
-import styles from './page.module.css'
+import styles from "./page.module.scss";
+import Header from "@/components/Header/Header";
+import FeatureLink from "@/components/FeatureLink/FeatureLink";
 
 export default function Home() {
   return (
```
Comment (Pihu01) [inline src/app/page.tsx:6]: Try to use arrow functions

File: src/app/page.tsx [hunk 2]
```diff
@@ -1,95 +1,16 @@
...
-        >
-          <h2>
-            Deploy <span>-&gt;</span>
-          </h2>
-          <p>
-            Instantly deploy your Next.js site to a shareable URL with Vercel.
-          </p>
-        </a>
+      <Header title="Magic Box" />
+      <div style={{ display: "grid", gridTemplateColumns: "1fr 1fr 1fr" }}>
```
Comment (Pihu01) [inline src/app/page.tsx:9]: do not use inline css separate in style file

File: src/components/Header/Header.tsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import HeaderStyles from "./Header.module.scss";
+
+interface HeaderProps {
+  title: string;
+}
+const Header: React.FC<HeaderProps> = ({ title }) => {
```
Comment (Pihu01) [inline src/components/Header/Header.tsx:7]: Instead React.FC you can use this way
const Header = ({title} :HeaderProps) =>{

 for reference read this blog - https://medium.com/raccoons-group/why-you-probably-shouldnt-use-react-fc-to-type-your-react-components-37ca1243dd13

File: src/components/TodoItems/TodoItems.tsx [hunk 4]
```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { TodoType } from "@/types";
+import { PiTrashSimpleThin } from "react-icons/pi";
+import styles from "./TodoItems.module.scss";
+
+const TodoItems = ({
+  todos,
+  todoDeleteHandler,
+}: {
```
Comment (Pihu01) [inline src/components/TodoItems/TodoItems.tsx:9]: If props are more than 1 create an interface instead inline types

File: src/app/page.tsx [hunk 5]
```diff
@@ -1,95 +1,16 @@
-import Image from 'next/image'
-import styles from './page.module.css'
+import styles from "./page.module.scss";
```
[the remark this answers is not in the document]
Comment (lovedeep5) [reply src/app/page.tsx:1]: Moved the import to end.

File: src/components/TodoItems/TodoItems.tsx (hunk 4 above)
Comment (lovedeep5) [reply src/components/TodoItems/TodoItems.tsx:9]: Done

**Pull request merged**